backend/controllers/prisonerProfile/prisonerCoursesQualificationsDetails.ts

```diff
@@ -0,0 +1,49 @@
+import { app } from '../../config'
+
+import { formatName, putLastNameFirst } from '../../utils'
+
+export default ({ prisonApi, esweService }) =>
+  async (req, res) => {
+    const { offenderNo } = req.params
+
+    type PrisonerDetails = {
+      offenderNo: string
+      firstName: string
+      lastName: string
+      bookingId: number
+      bookingNo: string
+      rootOffenderId: number
+      dateOfBirth: string
+      activeFlag: boolean
+      agencyId: string
+      assignedLivingUnitId: number
+    }
+
+    type coursesAndQuals = {
+      enabled: boolean
+      content: curious.LearnerEducationFullDetails[]
+    }
+
+    if (!app.esweEnabled) {
```

Comment:
I left this out in a previous education ticket and John S found that if he went directly to the URL he could bypass the flag completely (you wouldn't see any data because the EsweService stuff is also behind the flag) but you could get to an essentially blank page. So it's just in there to stop that. Will be removed when we take the flag out.

Reply:
Ok, fair enough